lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -30,11 +30,11 @@ module Billing #:nodoc:
     class OrbitalGateway < Gateway
       include Empty
 
-      API_VERSION = "5.6"
+      API_VERSION = "7.1"
 
       POST_HEADERS = {
         "MIME-Version" => "1.1",
-        "Content-Type" => "application/PTI56",
+        "Content-Type" => "application/PTI#{API_VERSION.gsub(/\./, '')}",
```

Comment:
> But they do not refer to the schema version as PTI71... This way seems marginally more readable

I figured as much, just thought I'd double check! I do think using the `7.1` as the `API_VERSION` is clearer for someone grok-ing the adapter so I'm ok with this. 

> though I suppose it's possible the gsub could break somehow...

Considering it's gsub-ing on a constant (well, as constant as constants get it Ruby...) I think the changes of it breaking are pretty close to zero!